api/src/main/java/org/apache/cloudstack/api/command/user/account/ListAccountsCmd.java

```diff
@@ -55,6 +61,12 @@
     @Parameter(name = ApiConstants.STATE, type = CommandType.STRING, description = "list accounts by state. Valid states are enabled, disabled, and locked.")
     private String state;
 
+    @Parameter(name = ApiConstants.DETAILS,
+               type = CommandType.LIST,
+               collectionType = CommandType.STRING,
+               description = "comma separated list of account details requested, value can be a list of [ all, resource, min]")
+    private List<String> viewDetails;
```

Comment:
details throughout CloudStack relates to an API arg that is of key:value map. Why not call something meaningful like `view` or verbosity etc?

Reply:
listDetails seems appropriate, as well

Reply:
@rhtyd @DaanHoogland 
this is same as viewDetails in ListVMsCmd and ListHostsCmd.java
In some other API, details are in key/value pairs.

I agree 'view' or 'listdetails' make more sense.
I am afraid of backward compatibility.

Reply:
You are right, @ustcweizhou . I think we can deprecate thiose and add better parameter-names there as well. It is not so much the internal name, 'viewDetails' as the API parameter name, ApiConstants.DETAILS (.equals("details")) which might be confusing as an external name, 
The proposal is to change the bit ```@Parameter(name = ApiConstants.DETAILS, ...```. As it is new I don't think we can have a compatibility issue there.
For my part this can not block this PR btw.

Reply:
@DaanHoogland good

> @ustcweizhou can you fix the merge conflict? Thanks.

@rhtyd rebased with latest master